### DISCUSSION TITLE
Is it possible to build a google colab for Lean Dojo?

### DISCUSSION THREAD

**Discussion by brando90** [lean-dojo/LeanDojo#70] [Q&A] [answered]
Due to the docker requirements and Lean, I was wondering if it is at all possible to build Lean Dojo in a Google Colab?

**Comment by yangky11** [comment]
The Docker requirement is primarily for Lean 3. I plan to drop it for Lean 4 but haven't got the capacity to do that yet. For Lean 4, you should feel free to set `CONTAINER=native` to run LeanDojo without Docker.

**Comment by yangky11** [comment, marked as answer]
Update: We have made "running w/o Docker" the default setting: https://github.com/lean-dojo/LeanDojo/pull/74. It should work out of the box for Lean 4. If you use LeanDojo with Lean 3, now you need to set the environment variable `CONTAINER` to `docker`.
